core/src/main/java/org/apache/calcite/tools/RelBuilder.java

```diff
@@ -1733,6 +1733,47 @@ public RelBuilder semiJoin(RexNode... conditions) {
     return semiJoin(ImmutableList.copyOf(conditions));
   }
 
+  /** Creates an AntiJoin: since, for the moment, there is no "AntiJoin" expression, the only
+   *  possibility to build an AntiJoin is using a LogicalCorrelate with SemiJoinType.ANTI */
```

Comment:
Can you make this into a comment that is useful to the user? Explain what an ant-join is. A SQL example works well. 

Maybe add a similar description to `semiJoin`.

Description of the limitations of the algebra can be moved to a non-javadoc comment.

Reply:
reviewed